soh/src/overlays/actors/ovl_Door_Ana/z_door_ana.c

```diff
@@ -139,6 +139,11 @@ void DoorAna_WaitOpen(DoorAna* this, GlobalContext* globalCtx) {
                 destinationIdx = this->actor.home.rot.z + 1;
             }
             globalCtx->nextEntranceIndex = entrances[destinationIdx];
+
+            if (gSaveContext.n64ddFlag) {
```

Comment:
do we want to do this always in rando? or just when doing entrance rando?

Reply:
This and some of the functions in z_player/z_play that determine the entrance override, I decided to not explicitly check for entrance rando settings.

The entrance override logic uses vanilla indices for anything not shuffled and allows us to use a modified scene table. We could put entrance skips (like skip child stealth/ganon tower) into the entrance logic to have one place for that kind of stuff.

But if we don't want to do that, then I understand.

Reply:
> We could put entrance skips (like skip child stealth/ganon tower) into the entrance logic to have one place for that kind of stuff.

We want this 👍